docs/sources/flow/release-notes.md

```diff
@@ -25,6 +25,28 @@ Other release notes for the different Grafana Agent variants are contained on se
 [release-notes-operator]: {{< relref "../operator/release-notes.md" >}}
 {{% /admonition %}}
 
+
+## v0.37
+
+### Breaking change: Renamed `non-indexed-labels` Loki processing stage to `structured_metadata`.
+
+It's necessary to renamed Loki processing stage definition `non_indexed_labels` to `structured_metadata` if this stage is used in Agent's config.
```

Comment:
```suggestion
If you use the Loki processing stage in your Agent configuration, you must rename the `non_indexed_labels` pipeline stage definition to `structured_metadata`.
```

We need to rephrase this sentence. Is this suggested text still technically accurate?